### ISSUE TITLE
I cannot login into Wok

### ISSUE THREAD

**Issue by iadegesso** [ptylenda/kimchi-docker#1]
Hi,
in my Gentoo-based system, I deployed this container from the command line as root, using the 'docker run -it -p 8001:8001 -d --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro ptylenda/kimchi-docker' command as told in the short readme.

The container starts but I cannot login in Wok/Kimchi since none of my system users is recognized (see the attachments).

Best regards,

Iade Gesso, PhD (in Computer Science)

![Selection_012](https://user-images.githubusercontent.com/5804075/56081381-069c4080-5e0d-11e9-9f64-0835974dbb84.jpg)
![Selection_013](https://user-images.githubusercontent.com/5804075/56081382-069c4080-5e0d-11e9-925d-78c978421eac.jpg)

**Comment by DRIgnazGortngschirl** [comment]
@iadegesso Same for me on a Ubuntu based system

**Comment by iadegesso** [comment]
@ptylenda any news?

**Comment by henrikp** [comment]
If anyone else wonders about the solution, then the answer is that the container is using its own permissions. I entered the container and changed the password for root.

**Comment by ghost** [comment]
After changing the pass and logging in, still doesn't seem to work for me - no templates tab?

**Comment by csrednicki** [comment]
Yeah, it is broken. Author didn't mention also about mounting two folders:
`/var/lib/kimchi/isos` and `/var/lib/libvirt/images`
